test/RPCSession.cpp

```diff
@@ -316,9 +316,7 @@ Json::Value RPCSession::rpcCall(string const& _methodName, vector<string> const&
 	request += "],\"id\":" + to_string(m_rpcSequence) + "}";
 	++m_rpcSequence;
 
-	BOOST_TEST_MESSAGE("Request: " + request);
```

Comment:
No, will remove it, but we should consider removing that in a separate PR. It produces gigantic clutter if you want to e.g. list which test cases are run (I found no way to distinguish between messages in a test case and the information about the test case) and we already have an option to at least show the call payload using `--show-messages`.